src/Microsoft.Extensions.DependencyModel/RuntimeLibrary.cs

```diff
@@ -18,8 +18,32 @@ public class RuntimeLibrary : Library
             IReadOnlyList<RuntimeAssetGroup> nativeLibraryGroups,
             IEnumerable<ResourceAssembly> resourceAssemblies,
             IEnumerable<Dependency> dependencies,
-            bool serviceable)
-            : base(type, name, version, hash, dependencies, serviceable)
+            bool serviceable) : this(
```

Comment:
(very minor nit): The format of calling ": this" and ": base" is inconsistent between this and the constructor below.  Below "base" is on its own line, and here "this" is on the same line as the last argument.